extension/packages/nextjs/package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "humanize-duration": "^3.28.0"
+  }
+}
```

Comment:
Think we are missing the type:

`"@types/humanize-duration": "^3",`

Reply:
Added!